tests/fixtures/scan-findings.jqpg

```diff
@@ -0,0 +1,25 @@
+jq -r results.jq
```

Comment:
How does this fixture get used?

Reply:
This is really just a test notebook I can/should remove it